code/modules/arcade/mob_hunt/mob_type_datums.dm

```diff
@@ -22,12 +22,11 @@
 					TYPE_GRASS,
 					TYPE_ICE,
 					TYPE_STEEL)
-	area_blacklist = list(/area/crew_quarters/toilet,
-						/area/crew_quarters/locker/locker_toilet,
-						/area/toxins/server_coldroom)
-	area_whitelist = list(/area/maintenance/turbine,
-						/area/maintenance/incinerator,
-						/area/crew_quarters/kitchen)
+	area_blacklist = list(/area/station/public/toilet,
```

Comment:
This whole file is a pain. I'll need to check if it checks by subtypes. This was also something I wanted to refactor to use variables on areas instead.

Reply:
This all seems to be in order